Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -1015,13 +1036,19 @@ class EndoscopyPathModel:
     - Add a single polyline
     """
 
-    def __init__(self, resampledCurve, inputCurve, outputPathNode=None):
+    def __init__(self, resampledCurve, inputCurve, cameraNode, outputPathNode=None, cursorType=None):
```

Comment:
The support for alternative cursor type has been removed, and this is not needed. See https://github.com/Slicer/Slicer/commit/cc0ee3744910abf96f1df78d0831bf98e20e25c7

Reply:
I will remove cursor-type support (but I will keep the outputPathNode support).

Reply:
To clarify, the commit referenced above is already integrated into `main`, this means that amending this pull request should be enough.